includes/Core/Util/Activation.php

```diff
@@ -186,21 +258,52 @@ private function get_activation_notice() {
 											); // phpcs:ignore WordPress.XSS.EscapeOutput.OutputNotEscaped
 											?>
 										</div>
-										<h3 class="googlesitekit-heading-1 googlesitekit-activation__title">
+										<h3 class="googlesitekit-heading-3 googlesitekit-activation__title">
```

Comment:
I'm with you on this change but we should check with @felixarntz about this first.

Reply:
@felixarntz Are you okay with this before/after? The existing size was really large and affecting usability I think; it throws the text onto multiple lines and is a bit too loud I think.

### Before
<img width="1137" alt="Screenshot 2019-10-24 18 50 07" src="https://user-images.githubusercontent.com/90871/67511625-3915d880-f68f-11e9-8d4e-22fbcb3245a3.png">

### After
<img width="1143" alt="Screenshot 2019-10-24 18 49 54" src="https://user-images.githubusercontent.com/90871/67511637-3d41f600-f68f-11e9-9ace-59387d67ce2f.png">

Reply:
I just talked through this with Mariya. Let's do the following:
* Keep the heading class at 3 like you changed it.
* Get rid of the grid in this notice entirely and move the `Start Setup` button below the heading, similar to how it is in the new setup screen you created in #642. The current UI of having the Setup button "randomly" (in the grid system) moved somewhere on the right is odd, and it's especially problematic having the button on the right and the checkbox on the left.
* Reduce the space above the checkbox and distribute the top-bottom space around the button more evenly.

I know this will unfortunately not decrease the height of this notice overall (because we move the button below), but it will make the UI easier to follow.

Reply:
I was thinking that the suggested changes here would break VRT but looking at the story we have I don't think it will because the component appears to be reimplemented here, rather than using a shared component in both places. Is this something we should update as well?

https://github.com/google/site-kit-wp/blob/fcb3431870f4827f8b81e2e4ab5e05da88fb7686/stories/wp-activation.stories.js#L14-L40

Reply:
Oh, that's probably a good call. 👍

Reply:
I updated these and the visual tests to match the new look.